bin/wmagent-resource-control

```diff
@@ -167,13 +167,12 @@ def printThresholds(resourceControl, desiredSiteName):
         print "%s - %d running, %d pending, %d running slots total, %d pending slots total%s:" % (
                         siteName, runningJobs, pendingJobs, runningSlots, pendingSlots, stateMsg)
 
-        for task in siteThresholds['thresholds']:
+        for task, thres in siteThresholds['thresholds'].iteritems():
```

Comment:
Interesting. I'll update this patch tomorrow. Still, depending on the length of the dictionary, it may be useful to keep iteritems() until the very last moment towards the python 3 migration